tensorboard/webapp/header/header_component.ts

```diff
@@ -54,6 +54,9 @@ import {Component} from '@angular/core';
       app-header-reload,
       settings-button {
         flex: 0 0 auto;
+        letter-spacing: -0.025em;
+        margin-left: 10px;
+        text-rendering: optimizeLegibility;
```

Comment:
Please apply this to only the `.brand` unless it is more correct to change other text like `.readme` and others.

Reply:
Whoops, of course; thanks!